addon/components/au-toggle-switch.hbs

```diff
@@ -15,5 +18,11 @@
     ...attributes
   />
   <span class="au-c-toggle-switch__toggle"></span>
-  <span class="au-c-toggle-switch__label">{{@label}}</span>
+  {{#if (has-block)}}
+    {{yield}}
```

Comment:
My intention was to align it a bit more with how `au-checkbox.hbs` & `au-radio.hbs` are done now:
- Firstly, allow block content.
- Secondly, deprecate `@label`.
- And lastly, remove `@label` from version `3.x.x` on so only block content can be used.

Does this seem like a good idea?

I think the possibility of passing block content is great to have. With that, cases like 'we need an info icon with a popover attached to it next to the label' are also covered. A bit more flexible then embedded loading state I guess?

In any case, I could still also wrap the `{{yield}}` within the `span.au-c-toggle-switch__label`.

Reply:
Indeed. But in the current version we can't really do that since the label styles would be missing, no? Adding the wrapper would be an option, but wouldn't that potentially affect the styles of the things you pass into the block (like the loader)?

Reply:
Maybe we could do the following:
- Adjust the styling & structure of the component and sync it with `au-checkbox` & `au-radio` within this PR: remove label wrapper (label text or block content would just exist directly inside the main element) & add basic 'label' styling on the main element (for that we would adjust the specific styling on the label within this component, but maybe thats a good thing, not sure why it is different anyway 🤷‍♂️). Additionally, we could deprecate `@label` in the meantime.
- When version `3.x.x` of `ember-appuniversum` is introduced, remove `@label` as an argument.

Or, alternatively, we could just add the loading state within this PR and bundle the rest of the changes (needed later on) for another PR.

Reply:
To illustrate the difference in styling regarding the labels, see below (the first one is `au-toggle-switch`, the second `au-checkbox`:

<img width="163" alt="CleanShot 2023-04-17 at 15 09 59@2x" src="https://user-images.githubusercontent.com/18174827/232493875-09e2459b-5c4c-4992-b919-c4a38b122d4e.png">
<img width="152" alt="CleanShot 2023-04-17 at 15 09 42@2x" src="https://user-images.githubusercontent.com/18174827/232493881-aac71687-3d84-45bb-b28b-08423ff5c718.png">

Reply:
I committed the remaining necessary changes ✅